gulp/ProjectSpecs/Patterns/Balloon.js

```diff
@@ -0,0 +1,13 @@
+/* 
+* Pattern Info
+**/
+const patternInfo = {
+    "codeName": "Balloon",
+    "inDevelopment": false,
+    "name": "Balloon",
+    "scss": "../scripts/OSFramework/OSUI/Feature/Balloon/scss/balloon",
+    "section": "Interaction"
```

Comment:
@BenOsodrac I'm not 100% sure about this section being Interaction. Could you explain the rationale?

Reply:
@gnbm I guess because a balloon will always imply some sort of interaction for it to appear, it's not like a Card (Content section), that just renders in the same way, flat on the screen.